corehq/apps/app_manager/forms.py

```diff
@@ -7,7 +7,9 @@
 
 
 class CopyApplicationForm(forms.Form):
-    domain = forms.CharField(label=_("Copy this app to project:"))
+    domain = forms.CharField(
+        label=_("Copy this app to project"),
+        widget=forms.TextInput(attrs={"data-bind":"typeahead: domain_names"}))
```

Comment:
Yay! Closed out http://manage.dimagi.com/default.asp?189482

Agree hr names would be better, but this accommodates superusers better.
This is more a superuser feature so I'm good with this implementation.

On Thu, Dec 17, 2015 at 5:36 PM, dimagimon <notifications@github.com> wrote:

> In corehq/apps/app_manager/forms.py
> <https://github.com/dimagi/commcare-hq/pull/9648#discussion_r47971181>:
>
> > @@ -7,7 +7,9 @@
> >
> >
> >  class CopyApplicationForm(forms.Form):
> > -    domain = forms.CharField(label=_("Copy this app to project:"))
> > +    domain = forms.CharField(
> > +        label=_("Copy this app to project"),
> > +        widget=forms.TextInput(attrs={"data-bind":"typeahead: domain_names"}))
>
> E231 missing whitespace after ':'
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/dimagi/commcare-hq/pull/9648/files#r47971181>.
>